### DISCUSSION TITLE
Downlink/Remote Control Help

### DISCUSSION THREAD

**Discussion by SionHuws99** [cyberman54/ESP32-Paxcounter#768] [Q&A] [answered]
Hi,

I am having trouble with the Remote Control feature on the TTN downlink.

If "Set display on/off" is 0x04 and display off = 0
How is this then entered into the downlink?

And also if sleep cycle = 0x19 and the data I enter will be in seconds? Where do I input the seconds in the hex? I want the sleep cycle to be 10 minutes which is 600 seconds.

I am new to all of this so please be patient. Thanks!

**Comment by cyberman54** [comment]
@SionHuws99 Did you find the example in section _remote control_ in `readme.md` ?

**Reply by SionHuws99** [reply]
Yes I understand that example because you enter the number and it returns the information. However I am confused as to how to enter the extra values  for those options with multiple choice for example the display on/off 0=off and 1=on. Also I am confused as to how I set the sleep timer where do I put the seconds value?

Thanks

**Reply by cyberman54** [reply]
You put the value after the command, that's all.
<command><value>

**Reply by SionHuws99** [reply]
So to turn display off its 04 00?

And to set sleep cycle

19 500?

**Reply by cyberman54** [reply]
use hex, not dec.

**Reply by SionHuws99** [reply]
The display doesn't change when I enter the values, could you please tell me what I should be entering to turn the display off and the sleep cycle to 10 minutes as my attempts don't seem to be working. Really sorry, I appreciate your help!

**Comment by t-huyeng** [comment]
It probably depends on how your applications server expects your downlink payload. I am using ChirpStack, so I need Base64 encoded payload strings.

- To turn the display off you need to send `0x04  0x00` (hex) or `BAA=` (Base64) to Port 2. 
- To turn the display back on send: `0x04  0x01`(hex)  or `BAE=` (Base64) to Port 2.

**Comment by SebastianKotstein** [comment, marked as answer]
It seems that the "set sleep cycle" (0x19) command must be complemented by the "store device configuration" (0x21) command. Otherwise, the paxcounter will revert to the initially/last configured sleep cycle (stored in NVRAM) after it wakes up from deep sleep. I also noticed that the new sleep cycle will not be applied immediately. Instead, it takes another sleep cycle with the old duration before the new settings are applied for all subsequent cycles.

To set the sleep cycle to 600 seconds (dec: 60, hex: 0x3C), use the following byte sequence (hex):
`0x19 0x00 0x3C 0x21`

**Reply by cyberman54** [reply]
yes, that's how it works.
